readthedocs/projects/models.py

```diff
@@ -201,6 +204,17 @@ class Project(models.Model):
             'DirectoryHTMLBuilder">More info on sphinx builders</a>.',
         ),
     )
+    urlconf = models.CharField(
+        _('Documentation URL Configuration'),
+        max_length=255,
+        default=None,
+        null=True,
+        help_text=_(
+            'Supports the following keys: $language, $version, $subproject, $filename. '
+            'An example `$language/$version/$filename`.'
```

Comment:
Not sure why `Default:` is better here? There isn't a default for this field.

Reply:
Because that URL is what Read the Docs uses by default: `/en/latest/filename.html`. It's not the default of the field, but that's the help text presented to the user.